src/main/groovy/com/profitbricks/sdk/Common.groovy

```diff
@@ -76,6 +77,9 @@ final class Common {
 
             while (true) {
                 final path = loc.toURL().path - URLParts.path
+                if (path.getAt(0) == '/') {
```

Comment:
If it does not get chopped the path will look like this `/cloudapi/v3//requests/01d43acc-fe50-417c-95cd-7c3a796612c4/status` and the request will fail due to the double slashes.